packages/orbit-components/src/Breadcrumbs/BreadcrumbsItem/index.tsx

```diff
@@ -1,63 +1,40 @@
 "use client";
 
 import * as React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import defaultTheme from "../../defaultTheme";
-import ChevronForward from "../../icons/ChevronForward";
 import type { Props } from "./types";
+import ChevronForward from "../../icons/ChevronForward";
 
-const StyledBreadcrumbsItem = styled.li`
-  display: flex;
-  align-items: center;
-`;
-
-const StyledBreadcrumbsItemAnchor = styled(({ component, children, isClickable, ...props }) => {
-  const Component = isClickable ? component : "div";
-  return <Component {...props}>{children}</Component>;
-})`
-  ${({ theme, $active, isClickable }) => css`
-    font-weight: ${$active ? theme.orbit.fontWeightBold : theme.orbit.fontWeightMedium};
-    color: ${theme.orbit.paletteInkDark};
-    text-decoration: ${isClickable && !$active ? "underline" : "none"};
-
-    ${isClickable &&
-    css`
-      transition: color ${theme.orbit.durationFast} ease-in-out;
-      cursor: pointer;
-      border: none;
-      padding: 0;
-      outline-offset: 1px;
-
-      &:hover {
-        text-decoration: none;
-        color: ${theme.orbit.paletteProductNormalHover};
-      }
-    `};
-  `}
-`;
-
-StyledBreadcrumbsItemAnchor.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledBreadcrumbsItemIcon = styled(ChevronForward)`
-  ${({ theme }) => css`
-    margin: 0 ${theme.orbit.spaceXXSmall};
-  `}
-`;
+const Component = ({
+  href,
+  onClick,
+  component: Comp,
+  children,
+  ...props
+}: Props &
+  Pick<
+    React.HTMLAttributes<HTMLDivElement | HTMLAnchorElement | HTMLButtonElement>,
+    "itemScope" | "itemType" | "itemProp" | "itemID" | "className"
+  >) => {
+  if (onClick == null || href == null) return <div {...props}>{children}</div>;
+  if (Comp != null) return <Comp {...props}>{children}</Comp>;
+  if (href == null)
```

Comment:
Line 22 is unreachable, no? If `href == null` it will return the `div` on line 20 and never reach line 22